librosa/core/audio.py

```diff
@@ -203,7 +203,7 @@ def to_mono(y):
 
 
 @cache
-def resample(y, orig_sr, target_sr, res_type='sinc_best', fix=True, **kwargs):
+def resample(y, orig_sr, target_sr, res_type='sinc_best', fix=True, scale=False, **kwargs):
```

Comment:
`scale=False` is to preserve backwards compatibility for user-facing functions.  Most resampling applications do not need to preserve energy scale if you interpret them as interpolating the original signal at a different rate.  Scale is only really important for comparing filter response before and after resampling, as we do in CQT.